src/main/java/seedu/ui/Ui.java

```diff
@@ -0,0 +1,19 @@
+package seedu.ui;
+
+public class Ui {
```

Comment:
Appreciate the improvements to UI and helping create a new class for UI.

Could maybe also include the message for help command into the class.